src/Components/dashboard/Dashboard.js

```diff
@@ -894,7 +696,24 @@ class Dashboard extends Component {
                   <SubscriptionIcon className="plc-w-10 plc-h-10 plc-pt-2 plc-pr-1 plc--ml-2" />
                 }
                 title={this.locale("labels.subscriptions")}
-                content={this.renderSubscriptions()}
+                content={
+                  <SubscriptionsMenu
```

Comment:
all the functions that the `SubscriptionsMenu` needs are being passed as props, this is because some of them are used by sections other than the subs section, so moving them would mean duplicating the logic in two different places, which I think shouldn't happen + there's a bit of coupling going on in the `Dashboard` component

Reply:
I get some of the props, but some don't make sense. 
`setView` for example, as we can use usePelcro.switchView inside.
onClose too, I guess it's just calling `resetView`, 

More methods that are already abstracted by usePelcro:
- setProductAndPlan
- setSubscriptionIdToRenew